plugins/woocommerce/src/Internal/Admin/WcPaySubscriptionsPage.php

```diff
@@ -74,6 +71,67 @@ public function register_subscriptions_page() {
 		}
 	}
 
+	/**
+	 * Returns true if the store is eligible for the WooCommerce subsciptions empty state experiment.
+	 *
+	 * @return bool
+	 */
+	private function is_store_experiment_eligible() {
+		// Ineligible if WooCommerce Payments OR an existing subscriptions plugin is installed.
+		$installed_plugins      = PluginsHelper::get_installed_plugin_slugs();
+		$plugin_ineligible_list = array(
+			'woocommerce-payments',
+			'woocommerce-subscriptions',
+			'subscriptio',
+			'subscriptions-for-woocommerce',
+			'subscriptions-for-woocommerce-pro',
+			'sumosubscriptions',
+			'yith-woocommerce-subscription',
+		);
+		foreach ( $plugin_ineligible_list as $plugin_slug ) {
+			if ( in_array( $plugin_slug, $installed_plugins, true ) ) {
+				return false;
+			}
+		}
+
+		// Ineligible if store address is not compatible with WCPay Subscriptions (US).
+		$store_base_location = wc_get_base_location();
+		if ( empty( $store_base_location['country'] ) || 'US' !== $store_base_location['country'] ) {
+			return false;
+		}
+
+		// Ineligible if store has not been active for at least 6 months.
+		if ( ! WCAdminHelper::is_wc_admin_active_in_date_range( 'month-6+' ) ) {
+			return false;
+		}
+
+		// Ineligible if store has not had any sales in the last 30 days.
+		if ( ! $this->get_store_recent_sales_eligibility() ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Returns true if the store has an order completed within the last 30 days.
+	 *
+	 * @return bool
+	 */
+	private function get_store_recent_sales_eligibility() {
+		// Get a single order that was marked as completed within the last 30 days.
+		$orders = wc_get_orders(
+			array(
+				'date_paid' => '>' . strtotime( '-30 days' ),
+				'status'    => array( 'wc-completed' ),
```

Comment:
We might be able to remove the status requirement here given we're checking that the order has a paid date, but I'm happy to discuss. As I see it, there are 2 main reasons we can remove it: 

1. The completed status is used to indicate orders that has been shipped or don't need shipping (virtual products). The original criteria mentioned a sale in the last 30 days so I don't think it necessarily needs to have a completed status. ie a sale could have a processing status for example. My dev site wasn't eligible initially because I don't have any completed orders (albeit my local isn't a legit store). 
2. I'm not sure how these queries work with custom statuses but stores that use a custom status to indicate the order is shipped or "finished" may not get caught by this. This I would need to confirm.

Reply:
Thanks for catching this one @james-allan!

Yeah, `wc-completed` is too strict and will risk missing any orders that may be "finished" by a method other than `wc-completed`.

`date_paid` > 30 days age will be sufficient for checking if the store has "had < 1 sale in last 30 days".

Reply:
I've removed the `wc-completed` status query argument in  48ead02  and updated the **Store sales ineligibility** test instructions in the PR description.